test/spaceSearch.spec.ts

```
@@ -14,6 +14,7 @@ import { SearchService } from '../src/services/SearchService';
 import { DayOfWeekLOSRule } from '../src/proto/lpms';
 import { FormattedDate } from '../src/services/DBService';
 import { convertDaysToSeconds } from '../src/utils';
+import { facility, space } from './common';
 
 describe('search service test', async () => {
   const facilityId = '0x1234567890';
```

This facility should throw as invalid `bytes32` string.